data/mods/ssb/moves.ts

```diff
@@ -451,6 +451,55 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Steel",
 	},
 
+	// Arcticblast
+	radiantburst: {
+		accuracy: true,
+		basePower: 10,
+		category: "Special",
+		desc: "Move causes Tapu Fini to become Brilliant if not, and vice versa. Move Mode depends on whether Tapu Fini is Brilliant or not.",
+		shortDesc: "Move's Mode depends on whether Tapu Fini has the Brilliant condition or not.",
+		name: "Radiant Burst",
+		pp: 16,
+		priority: 0,
+		flags: {protect: 1},
+		onTryMovePriority: 100,
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			if (source.volatiles['brilliant']) {
+				this.add('-anim', target, 'Recover', target);
+			} else {
+				this.add('-anim', source, 'Diamond Storm', target);
+			}
+		},
+		onModifyMove(move, pokemon) {
+			if (pokemon.volatiles['brilliant']) {
+				move.basePower = 180;
+				move.accuracy = 100;
+				move.type = "Fairy";
+				delete move.infiltrates;
+			}
```

Comment:
Can we do this the same way we do a move like future sight?

Reply:
i felt like this was the best way to do it

Reply:
yeah I think we should 100% go the futuremove check route

Reply:
we should also have the ingrain effect implanted instead of an additional volatile